server/methods/accounts/accounts.js

```diff
@@ -145,6 +146,35 @@ function validateAddress(address) {
  * Reaction Account Methods
  */
 Meteor.methods({
```

Comment:
So the style that we want to use for Meteor methods is to `export` them as individual method, and then just declare them as Meteor methods at the bottom of the file. This allows them to more easily be tested or used by other server-side code.